app/models/host_status/configuration_status.rb

```diff
@@ -93,6 +93,8 @@ def to_status(options = {})
     end
 
     def relevant?(options = {})
+      # Do not calculate global status from legacy configuration when plugin is present.
+      return false if Foreman::Plugin.installed?('foreman_host_reports')
```

Comment:
Looking at https://github.com/theforeman/foreman/blob/9b83dfe537c2eaab1e5f461cc4d77715fce9ad38/app/views/api/v2/hosts/main.json.rabl#L30 I thought that it will skip the `ConfigurationStatus`, but it seems that it doesn't for API...